spec/lib/amazon_product_api/search_response_spec.rb

```diff
@@ -43,4 +43,40 @@
       end
     end
   end
+
+  describe '#success' do
+    context 'a successfull response' do
+      it 'should return true for success codes' do
+        [ 200, 204, 302, 304 ].each do |code|
```

Comment:
Nit: Worth defining a method or `let` block to return `success_codes`?

Reply:
Will do